css/hl.css

```diff
@@ -3374,7 +3375,15 @@ a.badge:hover, a.badge:focus {
     margin-right: auto; }
   .thumbnail .caption {
     padding: 9px;
-    color: #333333; }
+    color: #333333;
```

Comment:
This is more a note for the future than for this PR, but we could definitely start using CSS variables for colors like this.